### ISSUE TITLE
can't uninstall or update 

### ISSUE THREAD

**Issue by measdot** [steelbrain/linter#1293]
Uninstalling “linter” failed
Failed to delete linter: EACCES: permission denied, unlink '/Users/kumar/.atom/packages/linter/

**Comment by Arcanemagus** [comment]
Did you happen to run `apm install` as `root` by accident?

The solution to this is to go into `/Users/kumar/.atom/packages/` and remove any folder not owned by your user, then re-install the packages affected by this by running `apm install package1 package2 ...` as your own user, *not* `root`.

**Comment by measdot** [comment]
@Arcanemagus Apologies for very late response of such a quick and accurate help.

To follow what you suggested I had to do following in the `/Users/<username>/.atom/packages/` directory:
- `ls -all` : _to check what directories are owned by root, in my case only linter I installed with sudo permissions_ 
- `find -user root -delete` : _to delete all files owned by root in the `packages` directory._
- `apm install linter` : to install linter again as my own user

**PS:** My current version of linter is 1.11.18 while the latest is 1.11.21 at the time of writing. Still Atom does not show the **1.1.21 update**  in its **Available Updates** list under **Updates** tab which was there before re-installing linter as my own user.
